src/crimson/mon/MonClient.cc

```diff
@@ -587,6 +587,11 @@ int Client::handle_auth_request(crimson::net::ConnectionRef con,
     logger().info("skipping challenge on {}", con);
     authorizer_challenge = nullptr;
   }
+  if (!active_con) {
+    logger().info("auth request during inactivity period");
+    // let's instruct the client to come back later
+    return -EBUSY;
```

Comment:
confirmed. see the caller site:

https://github.com/ceph/ceph/blob/c88bfc8bdd3fad5c60d87ba7a1a4ef9d3d220a72/src/crimson/net/ProtocolV2.cc#L992-L995

the caller of `_handle_auth_request()` just calls `close(false)` when seeing an exception.